src/commands/handlers/messaging.js

```diff
@@ -70,8 +70,9 @@ var handlers = {
 
         if ((message.charAt(0) === '\01') && (message.charAt(message.length - 1) === '\01')) {
             // CTCP request
-            if (message.substr(1, 6) === 'ACTION') {
-
+            message = message.slice(1, -1); // remove first and last characters (\01 both times)
+            var messageArray = message.split(" ");
```

Comment:
`messageArray` should be `message_array` and declared at the top of the function and using single quotes for consistency. Better yet, use `ctcp_command = message.slice(1, -1).split(' ')[0];` to be more precise.